pallets/passkey/src/weights.rs

```diff
@@ -42,59 +42,67 @@ pub struct SubstrateWeight<T>(PhantomData<T>);
 impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	/// Storage: `System::Account` (r:1 w:1)
 	/// Proof: `System::Account` (`max_values`: None, `max_size`: Some(128), added: 2603, mode: `MaxEncodedLen`)
+	/// Storage: `System::AllExtrinsicsLen` (r:1 w:0)
+	/// Proof: `System::AllExtrinsicsLen` (`max_values`: Some(1), `max_size`: Some(4), added: 499, mode: `MaxEncodedLen`)
 	/// Storage: `TransactionPayment::NextFeeMultiplier` (r:1 w:0)
 	/// Proof: `TransactionPayment::NextFeeMultiplier` (`max_values`: Some(1), `max_size`: Some(16), added: 511, mode: `MaxEncodedLen`)
 	fn validate() -> Weight {
 		// Proof Size summary in bytes:
-		//  Measured:  `178`
+		//  Measured:  `235`
 		//  Estimated: `5078`
-		// Minimum execution time: 1_002_000_000 picoseconds.
-		Weight::from_parts(1_011_000_000, 5078)
-			.saturating_add(T::DbWeight::get().reads(2_u64))
+		// Minimum execution time: 1_728_647_000 picoseconds.
+		Weight::from_parts(1_741_351_000, 5078)
+			.saturating_add(T::DbWeight::get().reads(3_u64))
 			.saturating_add(T::DbWeight::get().writes(1_u64))
 	}
 	/// Storage: `System::Account` (r:1 w:1)
 	/// Proof: `System::Account` (`max_values`: None, `max_size`: Some(128), added: 2603, mode: `MaxEncodedLen`)
+	/// Storage: `System::AllExtrinsicsLen` (r:1 w:1)
+	/// Proof: `System::AllExtrinsicsLen` (`max_values`: Some(1), `max_size`: Some(4), added: 499, mode: `MaxEncodedLen`)
 	/// Storage: `TransactionPayment::NextFeeMultiplier` (r:1 w:0)
 	/// Proof: `TransactionPayment::NextFeeMultiplier` (`max_values`: Some(1), `max_size`: Some(16), added: 511, mode: `MaxEncodedLen`)
 	fn pre_dispatch() -> Weight {
 		// Proof Size summary in bytes:
-		//  Measured:  `178`
+		//  Measured:  `131`
 		//  Estimated: `5078`
-		// Minimum execution time: 1_001_000_000 picoseconds.
-		Weight::from_parts(1_015_000_000, 5078)
-			.saturating_add(T::DbWeight::get().reads(2_u64))
-			.saturating_add(T::DbWeight::get().writes(1_u64))
+		// Minimum execution time: 1_745_366_000 picoseconds.
```

Comment:
@wilwade After running the weights in CI it looks like it is actually close to 2 billion instead if 1. Which means the transactions would be even more expensive.